### PR TITLE
Restructure tutorial

### PR DESCRIPTION
**Pull request by timgrossmann**
- removed the old tutorials (3-6)
- added Instruction panel to the tutorial levels
- modified the story dialog to only display goals and story
- added unlocked block to win dialog
- switched the direction to make it work with the right blocks

- added more tutorial parameters
  - instructions (img and text)
  - unlocked block in story modal (img and text)

> Should be ready to implement new improved tutorials now

###### It's probably smart to wait until after the user test this weekend to merge it

### CODE REVIEW

File: pages/tutorial.js [hunk 1]
```diff
@@ -77,11 +80,12 @@ const tutorialView = (state, prev, send) => {
...
-        gameHtml,
-        goalProgressHtml
-      ]
+        instructionHtml,
+        goalHtml
+      ],
+      blocklyHtml,
+      gameHtml
     ]
   })
```
Comment (H3rby7) [inline pages/tutorial.js]: Could you add a button to quickly hide/show the new panel with the instructions and the goal? (kind of like a slide-in/hamburger menu). As of state now the view feels really squished into the browser with 3 panels active.  
Yes I know they are resizable and the player can simply resize it out, but i really think the instruction part should be a slide-in. (Maybe upon hovering over the slide-in button we could display the goals again without opening the slide-in)  

image

**Review by H3rby7 — CHANGES_REQUESTED** [review]
Could you take a look at the comment for the instruction view?

File: pages/tutorial.js (hunk 1 above)
Comment (paulsonnentag) [reply pages/tutorial.js]: I agree with @H3rby7. I think the image is redundant. If you don't want to loose the arrows you could add them as entities into the actual game world.


What do you think about something like this. I know mad gimp skillz
44625eec-f5d5-11e6-92a2-9b6269d0a6a0 1

You could actually modify the page layout, so you can set the initial size of the panels. Something like this:

``` Javascript
  const panelGroupHtml = panelGroup.component(...context)(id, {
    props: {
      panelViews: panels
    }
    initialState: {
      panelSizes: [1, 2, 2]
    }
  })
```

in [pageLayout](https://github.com/robotopia-x/robotopia/blob/ad41215a5b473ecde0aad0b851cba558c21663bb/elements/page-layout.js#L20)

**Comment by timgrossmann** [comment]
👍 Will take care of it tomorrow

File: pages/tutorial.js (hunk 1 above)
Comment (timgrossmann) [reply pages/tutorial.js]: That looks like a nice idea! 
You should add that to the component, I seriously hardly understand something in the panel-group file. (also there is very few documentation besides the default behaviour of that module)

I adjusted the instruction panel to look like you suggested.

**Comment by timgrossmann** [comment]
I don't know why, but the toolbox is always empty since rebasing it on develop.

**Comment by paulsonnentag** [comment]
The resize doesn't work yet, I'll fix it tomorrow

**Pull request merged**